pybamm/__init__.py

```diff
@@ -174,8 +179,13 @@ def version(formatted=False):
     Geometry1DMacro,
     Geometry3DMacro,
     Geometry1p1DMacro,
+    Geometry2p1DMacro,
     Geometry1DMicro,
     Geometry1p1DMicro,
+    Geometry1p0p1DMicro,
+    Geometry1p1p1DMicro,
+    Geometry2p0p1DMicro,
+    Geometry2p1p1DMicro,
```

Comment:
given how many of these Geometry classes we now have, we might want to merge them into a fewer classes with parameters referring to the various subdimensions